crates/rover-client/src/operations/graph/introspect/schema.rs

```diff
@@ -379,26 +392,26 @@ mod tests {
         assert_eq!(
             schema.encode(),
             indoc! { r#"
-        directive @cacheControl(maxAge: Int, scope: CacheControlScope) on FIELD_DEFINITION | OBJECT | INTERFACE
-        "Exposes a URL that specifies the behaviour of this scalar."
-        directive @specifiedBy("The URL that specifies the behaviour of this scalar." url: String!) on SCALAR
+        "The `Upload` scalar type represents a file upload."
+        scalar Upload
         type Query {
           "A simple type for getting started!"
           hello: String
           cats(cat: [String]! = ["Nori"]): [String]!
         }
+        enum CacheControlScope {
+          PUBLIC
+          PRIVATE
+        }
         input BooleanQueryOperatorInput {
           eq: Boolean
           ne: Boolean
           in: [Boolean]
           nin: [Boolean]
         }
-        enum CacheControlScope {
-          PUBLIC
-          PRIVATE
-        }
-        "The `Upload` scalar type represents a file upload."
-        scalar Upload
+        directive @cacheControl(maxAge: Int, scope: CacheControlScope) on FIELD_DEFINITION | OBJECT | INTERFACE
+        "Exposes a URL that specifies the behaviour of this scalar."
+        directive @specifiedBy("The URL that specifies the behaviour of this scalar." url: String!) on SCALAR
     "#}
```

Comment:
I am not entirely sure how significant the order is in the introspection SDL. Can we double check if the spec defines a specific order (I don't think I've seen this, but I might be wrong!), and check with someone who uses the introspection feature on their expectations?

Reply:
I will check, but as far as I recall there is not specific order in the specs. But if there is one I think I followed the same order than made in the grammar. I will double check :)